flink-clients/src/main/java/org/apache/flink/client/cli/DynamicPropertiesUtil.java

```diff
@@ -49,7 +49,7 @@ class DynamicPropertiesUtil {
      * Parses dynamic properties from the given {@link CommandLine} and sets them on the {@link
      * Configuration}.
      */
-    static void encodeDynamicProperties(
+    static Configuration encodeDynamicProperties(
```

Comment:
I think we don't need this change. The dynamic properties are parsed and inserted to the configuration passed-in.